tutorials/solids/thermoelasticity/hotSphere/README.md

```diff
@@ -1,5 +1,8 @@
 ---
 sort: 1
+output:
+  html_document:
+    df_print: paged
```

Comment:
@ztukovic : I guess this option changes how the README is displayed. Do you know what this is needed for?

Reply:
Sorry I have opened this file with RStudio and made these modification unintentionally. You can restore old version

Reply:
Ah, OK. I will revert it.

... maybe this helps in some way: I will check this some other day.